source/documentation/data-docs/curated-databases-docs/databases-for-analysis-and-apps.md

```diff
@@ -0,0 +1,56 @@
+
+- A good general rule is to do any aggregates, joins filtering in SQL on our databases (using Athena) first and then bring the resulting table into R/Python for more nuanced analytical calculations / transforms. If you just need to aggregate, join or filter your data and not do anything special with it then it might be worth considering doing all of your transforms using SQL Athena.
+
+- Is the data you are querying large? If so you might want to consider aggregating it or only selecting the columns you need using SQL (Athena) first before you read the data into an R/Python dataframe. R/Python are going to read that data into memory so that at some point of the dataset is too large is will kill your AP instance (on the flip side Athena takes that load for you so better to use Athena for big table queries and joins).
+
+- If you need to do joins across datasets it is most likely that Athena is going to be much faster than Python/R unless your datasets are very small. So if you are doing some analysis with many joins use Athena.
+
+If in doubt about best practices we have slack channels for R, python, SQL, etc where you can ask.
+
+## Guidance on using databases / data for deployed apps
+
+This section of the guidance covers best practice for using data on the platform for deployed apps. If you want to know how to set up an app bucket go to the [deploy an R shiny app section](#deploying-an-r-shiny-app) of the guidance.
+
+### What to consider when giving an app access to data
+
+This is because apps are typically accessed by users without Analytical Platform accounts. To add additional security for our data, we like to restrict how much data apps can access, in order to prevent unwanted data leaks in the unlikely event of an app being compromised. In order to describe how this affects things when we give apps access to databases, we need to briefly touch on 'roles' within the Analytical Platform.
+
+When you access things on the Analytical Platform, you do so by taking on your 'role', which lists the set of things that you are permitted to do. This is how you interact with RStudio, Jupyter, Airflow, S3 buckets, etc. Since apps also need permissions to access things, they also have roles. When app users are interacting with the app, it will be carrying out tasks using its own role. In order to minimise the damage that an app could do in the unlikely event of it being compromised, we provide apps with much more restricted roles than those given to users. What this means is:
+
+- Apps cannot access (read/write) S3 buckets created by the AP users (instead they have their own 'app buckets' prefixed with `alpha-app-`)
+- Apps cannot access databases created for the AP users (these are all of the "curated" databases created by the Data Engineering Team)
```

Comment:
I can't believe approved house style is:
"curated" databases
Just curated databases, I think.